.github/workflows/release.yml

```diff
@@ -55,3 +55,64 @@ jobs:
         uses: pypa/gh-action-pypi-publish@897895f1e160c830e369f9779632ebc134688e1b
         with:
           repository-url: https://upload.pypi.org/legacy/
+
+  publish-docker-operator:
+    needs:
+      - packages
+    runs-on: ubuntu-latest
+    permissions:
+      attestations: write
+      id-token: write
+      contents: write
+    env:
+      DOCKER_IMAGE_NAME: docker.elastic.co/observability/elastic-otel-python-operator
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@988b5a0280414f521da01fcc63a27aeeb4b104db # v3.6.1
+
+      - name: Log in to the Elastic Container registry
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        with:
+          registry: ${{ secrets.ELASTIC_DOCKER_REGISTRY }}
+          username: ${{ secrets.ELASTIC_DOCKER_USERNAME }}
+          password: ${{ secrets.ELASTIC_DOCKER_PASSWORD }}
+
+      - uses: actions/download-artifact@v4
+        with:
+          name: packages
+          path: dist
+
+      - name: Extract metadata (tags, labels)
+        id: docker-meta
+        uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81  # v5.5.1
+        with:
+          images: ${{ env.DOCKER_IMAGE_NAME }}
+          tags: |
+            type=raw,value=latest,prefix=test-,enable={{is_default_branch}}
+            type=semver,pattern={{version}}
+          labels:
+            org.opencontainers.image.vendor=Elastic
+            org.opencontainers.image.title=elastic-otel-python-operator
+            org.opencontainers.image.description=EDOT Python image for Kubernetes OpenTelemetry Operator
+
+      - name: Build and push image
+        id: docker-push
+        uses: docker/build-push-action@5cd11c3a4ced054e52742c5fd54dca954e0edd85  # v6.7.0
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
```

Comment:
Is this building the image two times?

Reply:
Yes, that is my understanding. It will build once for each platform. The "multi-platform" image that is published is actually:  one image for each platform, plus an additional image "manifest" (this is the thing with the final `docker.elastic.co/observability/elastic-otel-python-operator:$TAG` name).